specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/AuthConfigs.json

```diff
@@ -430,6 +434,35 @@
         }
       }
     },
+    "TokenStore": {
+      "description": "The configuration settings of the token store.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "<code>true</code> to durably store platform-specific security tokens that are obtained during login flows; otherwise, <code>false</code>.\n The default is <code>false</code>.",
+          "type": "boolean"
+        },
+        "tokenRefreshExtensionHours": {
+          "format": "double",
+          "description": "The number of hours after session token expiration that a session token can be used to\ncall the token refresh API. The default is 72 hours.",
+          "type": "number"
+        },
+        "azureBlobStorage": {
+          "$ref": "#/definitions/BlobStorageTokenStore",
+          "description": "The configuration settings of the storage of the tokens if blob storage is used."
+        }
+      }
+    },
+    "BlobStorageTokenStore": {
+      "description": "The configuration settings of the storage of the tokens if blob storage is used.",
+      "type": "object",
+      "properties": {
+        "sasUrlSettingName": {
```

Comment:
for now, we don't support filesystemtokenStore, so we added validation in our code that user must set this property. But in future, it will become optional.

Reply:
fixed